<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>86</width>
    <height>100</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>5</width>
    <height>5</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>187</width>
    <height>150</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string extracomment="logo"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="minimumSize">
       <size>
        <width>86</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>5</width>
        <height>5</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>187</width>
        <height>150</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="whatsThis">
       <string extracomment="logo"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="pix">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>492</width>
           <height>333</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/img/robomotion.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Odczyt">
       <attribute name="title">
        <string>Odczyt</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Połączenie</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonSearch">
             <property name="text">
              <string>Wybierz Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonConnect">
             <property name="text">
              <string>Połącz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonCloseConnection">
             <property name="text">
              <string>Rozłącz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxDevices"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Sterowanie</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonLedOn">
             <property name="text">
              <string>LED off</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButtonREAD">
             <property name="text">
              <string>Odczytaj kartę </string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lbl_button_state">
             <property name="text">
              <string>Nieaktywny</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonLedOff">
             <property name="text">
              <string>LED on</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Kopiowanie UID zawodnika</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPushButton" name="copyToClipboard">
             <property name="text">
              <string>Skopiuj tag UID do schowka</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTextEdit" name="textEditLogs"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Zapis">
       <attribute name="title">
        <string>Nowy UID</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Włączenie zapisu NFC</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="SaveON">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Zapis do karty</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelsave">
             <property name="text">
              <string>Nieaktywny</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Komunikaty</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="0">
            <widget class="QTextEdit" name="textEditLogs_2"/>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>13777231</height>
              </size>
             </property>
             <property name="title">
              <string>Wpisz tag UID zawodnika</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="savenfc">
                <property name="text">
                 <string>zapisz</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Czas">
       <attribute name="title">
        <string>Odczyt czasu z bramek</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="2" column="0">
           <widget class="QTextEdit" name="textEditLogs_3"/>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>Bramka podwójna</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButtonDualGate">
               <property name="text">
                <string>Odczytywanie czasu z podwójnej  bramki</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="lbl_buton_dual_gate">
               <property name="text">
                <string>Nieaktywny</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>Bramka pojedyńcza</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="1">
              <widget class="QLabel" name="lbl_buton_one_gate">
               <property name="text">
                <string>Nieaktywny</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButtonOneGate">
               <property name="text">
                <string>Odczytywanie czasu z pojedyńczej bramki</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QPushButton" name="pushButtonCopyTime">
               <property name="text">
                <string>Skopiuj czas przejazdu do schowka </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
